skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/auth/AuthorizeAnsattSakTilgang.kt

```diff
@@ -6,64 +6,41 @@ import io.ktor.server.auth.*
 import io.ktor.server.response.*
 import io.ktor.server.util.*
 import io.ktor.util.*
-import kotlinx.coroutines.async
 import kotlinx.coroutines.coroutineScope
 import no.nav.pensjon.brev.skribenten.auth.AuthorizeAnsattSakTilgang.NAME
 import no.nav.pensjon.brev.skribenten.auth.AuthorizeAnsattSakTilgang.SAKSID_PARAM
-import no.nav.pensjon.brev.skribenten.auth.AuthorizeAnsattSakTilgang.enheterKey
 import no.nav.pensjon.brev.skribenten.auth.AuthorizeAnsattSakTilgang.sakKey
 import no.nav.pensjon.brev.skribenten.principal
-import no.nav.pensjon.brev.skribenten.services.NAVEnhet
-import no.nav.pensjon.brev.skribenten.services.NavansattService
-import no.nav.pensjon.brev.skribenten.services.PdlService
-import no.nav.pensjon.brev.skribenten.services.PenService
+import no.nav.pensjon.brev.skribenten.services.*
 import no.nav.pensjon.brev.skribenten.services.PenService.SakSelection
-import no.nav.pensjon.brev.skribenten.services.ServiceResult
 import org.slf4j.LoggerFactory
 
 object AuthorizeAnsattSakTilgang {
     const val NAME = "AuthorizeAnsattSakTilgang"
     const val SAKSID_PARAM = "saksId"
     val sakKey = AttributeKey<SakSelection>("AuthorizeAnsattSakTilgang:sak")
-    val enheterKey = AttributeKey<List<NAVEnhet>>("AuthorizeAnsattSakTilgang:enheter")
 }
 
 private val logger = LoggerFactory.getLogger(AuthorizeAnsattSakTilgang::class.java)
 
 @Suppress("FunctionName")
 fun AuthorizeAnsattSakTilgang(
-    navansattService: NavansattService,
     pdlService: PdlService,
     penService: PenService,
 ) = createRouteScopedPlugin(NAME) {
     on(AuthenticationChecked) { call ->
         coroutineScope {
             val principal = call.principal()
             val saksId = call.parameters.getOrFail(SAKSID_PARAM)
-            val navIdent = principal.navIdent
 
-            val sakDeferred = async { penService.hentSak(call, saksId) }
-            val navansattEnheterDeferred = async { navansattService.hentNavAnsattEnhetListe(call, navIdent) }
-
-            val ikkeTilgang = sakDeferred.await().map { sak ->
+            val ikkeTilgang = penService.hentSak(call, saksId).map { sak ->
```

Comment:
Er dette bare opprydning etter vi fjernet enhets-sjekk?

Reply:
Opprydding. Enhetssjekk ble fjernet for en tid tilbake, men henting av enhetsliste ble ikke fjernet. Vi trenger ikke å hente den listen for annet enn bestilling av brev nå.